src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -404,8 +425,10 @@ private async Task<bool> ValidateCredentialsWork(InputArguments input, ICredenti
                 }
                 catch (Exception ex)
                 {
-                    _context.Trace.WriteLine($"Failed to validate existing credentials using OAuth");
+                    var message = "Failed to validate existing credentials using OAuth";
+                    _context.Trace.WriteLine(message);
                     _context.Trace.WriteException(ex);
+                    _context.Trace2.WriteError(message);
```

Comment:
I agree here, let's NOT serialize the exception onto the end of the trace2 error message.